### ISSUE TITLE
Link to list of IIS Server Variables is dead.

### ISSUE THREAD

**Issue by rossmpersonal** [MicrosoftDocs/iis-docs#522]
At https://docs.microsoft.com/en-us/iis/get-started/whats-new-in-iis-10-version-1709/new-features-introduced-in-iis-10-1709 , the link to the list of IIS Server Variables is dead.

**Comment by lextm** [comment]
Seems to work now.

**Comment by rossmpersonal** [comment]
Yes. It does appear to work now. I don't know why it did work at the time I opened the ticket. Please close ticket. Thank you.
